desktop/npm-helper.js

```diff
@@ -392,7 +511,7 @@ const iconMeta_ = {
 ${
   // eslint really doesn't understand embedded backticks
 /* eslint-disable */
-Object.keys(icons).sort().map(name => {
```

Comment:
this is sorted by grid already so lets not lose that information. Else the charCode will be out of order which is harder to debug